aspnetcore/fundamentals/index.md

```diff
@@ -208,3 +208,7 @@ For more information, see [Choosing between .NET Core and .NET Framework](/dotne
 ## Choose between ASP.NET Core and ASP.NET
 
 For more information on choosing between ASP.NET Core and ASP.NET, see [Choose between ASP.NET Core and ASP.NET](xref:fundamentals/choose-between-aspnet-and-aspnetcore).
+
+## Making HTTP requests
+
+For information about using `HttpClientFactory` to access HttpClient instances to make HTTP requests, see [Making HTTP requests](xref:fundamentals/http-requests).
```

Comment:
Update the title in the link text to the suggested title (e.g., "Initiate HTTP requests").